cmake/templates/NSIS.template.in

```diff
@@ -836,16 +836,24 @@ Function PostInstallOptionsPage
     !insertmacro SetInstallOption $DesktopClientCheckbox @CLIENT_DESKTOP_SHORTCUT_REG_KEY@ ${BST_CHECKED}
   ${EndIf}
 
+  ; FIXME: Re-enable or permanently remove system tray "Console" for Interface-only installs.
+  ; ${If} @SERVER_COMPONENT_CONDITIONAL@
+  ;   ${NSD_CreateCheckbox} 0 $CurrentOffset$OffsetUnits 100% 10u "&Create a desktop shortcut for @SANDBOX_HF_SHORTCUT_NAME@"
+  ; ${Else}
+  ;   ${NSD_CreateCheckbox} 0 $CurrentOffset$OffsetUnits 100% 10u "&Create a desktop shortcut for @CONSOLE_HF_SHORTCUT_NAME@"
+  ; ${EndIf}
+  ;
+  ; Pop $DesktopConsoleCheckbox
+  ; IntOp $CurrentOffset $CurrentOffset + 15  
+  ; ; set the checkbox state depending on what is present in the registry
+  ; !insertmacro SetInstallOption $DesktopConsoleCheckbox @CONSOLE_DESKTOP_SHORTCUT_REG_KEY@ ${BST_UNCHECKED}
```

Comment:
If we're removing things, why is there a whole bunch of  commented out code being added? Did something go wrong during a merge?